### PR TITLE
Add ARM64 binary and package builds

### LINKED ISSUE #453: Sidecar installation in Ubuntu Docker container on macOS ARM
During the test-day for 5.0 I tried to install the sidecar into an Ubuntu Docker container on my M1 ARM Mac.

The sidecar is built for `armv7` which is 32bit and might not work on every 64bit ARM platform. The binaries for darwin/arm64 are for macOS, not for Linux.
Docker on M1/M2 outputs `aarch64` as the platform, which is not `arm64` and already created issues with the reporting binaries.

I could install the binaries by manually downloading the `.deb` and do the following:
```
dpkg --add-architecture armv7
dpkg -i graylog-sidecar_1.2.0-1_armv7.deb
``` 

I think an additional `arm64` build is a good addition, which hopefully installs without adding an architecture. If not, the discrepancy between `aarch64` and `arm64` somehow needs to get resolved.

### PR DESCRIPTION
**Pull request by bernd**
Fixes #453

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,7 +39,7 @@ test: ## Run tests
 build: ## Build sidecar binary for local target system
 	$(GO) build $(BUILD_OPTS) -v -o graylog-sidecar
 
-build-all: build-linux-armv7 build-linux build-linux32 build-windows build-windows32 build-darwin build-darwin-arm64 build-freebsd
+build-all: build-linux-armv7 build-linux-arm64 build-linux build-linux32 build-windows build-windows32 build-darwin build-darwin-arm64 build-freebsd
 
 build-linux: ## Build sidecar binary for Linux
```
Comment (mpfz0r) [inline Makefile:44]: How about we rename this for more clarity?
```suggestion
build-linux-amd64: ## Build sidecar binary for Linux-amd64
```

File: changelog/unreleased/pr-480.toml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+type = "a"
+message = "Add operating system packages for ARM64."
```
Comment (mpfz0r) [inline changelog/unreleased/pr-480.toml:2]:
```suggestion
message = "Add operating system packages for Linux ARM64."
```

**Review by mpfz0r — CHANGES_REQUESTED** [review]
LGTM, a few nitpicks

File: Makefile (hunk 1 above)
Comment (bernd) [reply Makefile]: @mpfz0r I did a larger cleanup of the Makefile. Please check! :slightly_smiling_face:

**Review by mpfz0r — APPROVED** [review]
LGTM

**Pull request merged**